src/geojson/FeatureCollection.cpp

```diff
@@ -242,11 +242,14 @@ void FeatureCollection::set_ids(const std::string& id_field_name) {
     this->update_ids();
 }
 
-void FeatureCollection::update_ids() {
+void FeatureCollection::update_ids(const std::string& alt_id ) {
     feature_by_id.clear();
     for (auto feature : features) {
         if (feature->get_id() != "") {
             feature_by_id.emplace(feature->get_id(), feature);
+            if( ~alt_id.empty() && (feature->get_id() != feature->get_id(alt_id)) ){
```

Comment:
They're effectively the same in this case, but do have different semantics: probably should have `!` instead of `~`

Reply:
Not real sure how/why I wrote it this way 😕 I'll fix that real quick

Reply:
fixed in [eb0ee18](https://github.com/NOAA-OWP/ngen/pull/665/commits/eb0ee18bfb9da2b2d656fc465d3cad22f04d0e4f)